pkg/libvirt/resolve_vm.go

```diff
@@ -99,3 +112,62 @@ func addToNotExistCache(pid uint64) {
 	}
 	cacheNotExist[pid] = false
 }
+
+func getMetadataVMID(vmid string) (string, error) {
+	metadataURI := config.GetLibvirtMetadataURI()
+	token := config.GetLibvirtMetadataToken()
```

Comment:
can we move to close to the usage?

Reply:
Thanks for your review. I will include all of your feedback  in the next iteration. 
However I don't understand this question. Could you please elaborate ?

Reply:
Sorry, I wasn't clear :( 
What I meant is that as a good practice we should keep variables close to its usage. E.g. `token` is initialised here but used may be 20 lines below. For a reader / maintainer look at the code, they have to keep a context of this when they read its usage 20 lines later.  But if the variables are moved closer to its actual usage, it reduces the cognitive load.

Reply:
Oh ok I see. I'm a long year C developer some habits are hard to get rid of !
I'm moving the variable close to its usage.

Reply:
Ahem ahem :)